runtime/env-defaults

```diff
@@ -18,3 +18,7 @@ DB_USER=${DB_USER:-}
 DB_PASS=${DB_PASS:-}
 
 DB_UNACCENT=${DB_UNACCENT:-false}
+
+DB_POSTGIS=${DB_POSTGIS:-false}
+DB_POSTGIS_HSTORE=${DB_POSTGIS_HSTORE:-true}
+DB_POSTGIS_TOPOLOGY=${DB_POSTGIS_TOPOLOGY:-true}
```

Comment:
I cannot see where `DB_POSTGIS_HSTORE` and `DB_POSTGIS_TOPOLOGY` variables are used? Did you intend to commit these variables?